src/services/platform/matrix/agent/matrix.agent.service.ts

```diff
@@ -195,34 +196,85 @@ export class MatrixAgentService {
 
   async messageCommunity(
     matrixAgent: IMatrixAgent,
-    msgRequest: MatrixAgentMessageRequestCommunity
+    messageRequest: MatrixAgentMessageRequestCommunity
   ): Promise<string> {
     const rooms = await matrixAgent.matrixClient.getGroupRooms(
-      msgRequest.communityId
+      messageRequest.communityId
     );
     const room = rooms.chunk[0];
 
     if (!room) {
       throw new Error('The community does not have a default room set');
     }
 
-    await this.message(matrixAgent, room.room_id, { text: msgRequest.text });
+    await this.message(matrixAgent, room.room_id, {
+      text: messageRequest.text,
+    });
 
     return room.room_id;
   }
 
   async message(
```

Comment:
We probably should rename this to sendMessage since we have deleteMessage and editMessage

Reply:
Good point - addressed in 896940843be935910f4036fba30e554b1e1411ad

Note: also removed a method in the same file that is not used - please check.